corehq/apps/hqcase/export.py

```diff
@@ -6,9 +6,13 @@
 def export_cases(domain, cases, workbook, filter_group=None, users=None, all_groups=None, process=None):
     by_user_id = dict([(user.user_id, user) for user in users]) if users else {}
     by_group_id = dict([(g.get_id, g) for g in all_groups]) if all_groups else {}
+
+    owner_ids = set(by_user_id.keys())
     if filter_group:
```

Comment:
is `filter_group` always non-False? it seems like this code would have crashed before this change if not since `owner_ids` is assumed to exist later